src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/DocumentProcessor.java

```diff
@@ -44,10 +45,18 @@ public void uploadPdfFiles(List<Pdf> pdfs, List<ScannableItem> scannedItems) {
             );
 
         if (filesWithoutUrl.isEmpty()) {
-            scannedItems.forEach(item -> item.setDocumentUrl(response.get(item.getFileName())));
+            scannedItems.forEach(item -> {
+                item.setDocumentUrl(response.get(item.getFileName()));
+                item.setDocumentUuid(extractDocumentUuid(response.get(item.getFileName())));
+            });
             scannableItemRepository.saveAll(scannedItems);
         } else {
             throw new DocumentUrlNotRetrievedException(filesWithoutUrl);
         }
     }
+
+    private String extractDocumentUuid(String documentUrl) {
+        //text after the last '/' in the url
+        return StringUtils.substringAfterLast(documentUrl, "/");
```

Comment:
could you add a sample url here ? easier for anyone looking at the code

Reply:
done